DEVELOPMENT.md

```diff
@@ -77,15 +77,30 @@ real life testing.
 
 ## Publishing a new kured release
 
-This is the most straight-forward in the whole process.
+Check that `README.md` has an updated compatibility matrix and that the
+url in the `kubectl` incantation (under "Installation") is updated to the
+new version you want to release.
+
+Now create the `kured-<release>-dockerhub.yaml` for e.g. `1.3.0`:
+
+```sh
+VERSION=1.3.0
+MANIFEST="kured-$VERSION-dockerhub.yaml"
+cat kured-rbac.yaml > "$MANIFEST"
+cat kured-ds.yaml >> "$MANIFEST"
+sed -i "s#docker.io/weaveworks/kured#docker.io/weaveworks/kured:$VERSION#g" "kured-$VERSION-dockerhub.yaml"
```

Comment:
You used the old(er) cached version of my suggestion, using `$MANIFEST` here is probably what you want.